notify_login_email_changed/views/email_templates.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="email_template_notify_email_change" model="mail.template">
+        <field name="name">Email Changed</field>
+        <field name="email_from">${ctx['new_email']|safe}</field>
+        <field name="email_to">${ctx['old_email']|safe}</field>
```

Comment:
The email should be sent to the old email of security reason.
The purchase is to let user notice if someone changes their email address.

Reply:
`<field name="email_to">${ctx['old_email']|safe}</field>` -> it is old email

Reply:
Sorry that I have overlooked the fields and thanks for the explanation.